data-export-bulk-opentargets/templated-conda-env.yaml

```diff
@@ -0,0 +1,12 @@
+name: data-export-bulk-r-base
+channels:
+  - defaults
+  - conda-forge
+  - bioconda
+  - ebi-gene-expression-group
+dependencies:
+  - opentargets-validator
```

Comment:
shouldn't we pin the ot validator to a specific version?

Reply:
Yes, probably. I'll set to the most recent, but let me know if it should be pinned back